### PR TITLE
docs: fix spelling issues

### PR DESCRIPTION
**Pull request by nnsW3**
Hello
While looking through your docs I found and fixed several spelling issues.
Br, Elias.

### CODE REVIEW

File: documentation/src/crates/primitives/bits.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ The `AsRef` and `Deref` traits from `derive_more` crate are derived for both of
 
 The `Arbitrary` trait from the `arbitrary` crate and the `PropTestArbitrary` trait from `proptest_derive` crate are derived conditionally when either testing or the "arbitrary" feature is enabled. 
 
-The code also provides conversions between `B256`, `Address` and various other types such as `u64`, `primitive_types::H256`, `primitive_types::H160`, `primitive_types::U256`, and `ruint::aliases::U256`. The `impl` From blocks specify how to convert from one type to another.
+The code also provides conversions between `B256`, `Address` and various other types such as `u64`, `primitive_types::H256`, `primitive_types::H160`, `primitive_types::U256`, and `ruint::aliases::U256`. The `impl` From blocks specifies how to convert from one type to another.
```
Comment (onbjerg) [inline documentation/src/crates/primitives/bits.md:11]: this is actually incorrect, the blocks being referred to here are code blocks

File: documentation/src/crates/primitives/kzg.md [hunk 2]
```diff
@@ -6,7 +6,7 @@ This module houses;
 
 1. `KzgSettings`: Stores the setup and parameters needed for computing and verify KZG proofs.
 
-    The `KZG` premitive provides a default `KZGSettings` obtained from [this]( https://ceremony.ethereum.org/) trusted setup ceremony, a provision is also made for using a custom `KZGSettings` if need be, this is available in the `env.cfg`.
+    The `KZG` primitive provides a default `KZGSettings` obtained from [this]( https://ceremony.ethereum.org/) trusted setup ceremony, a provision is also made for using a custom `KZGSettings` if need be, this is available in the `env.cfg`.
 
 
-2. `trusted_setup_points`: This module contains functions and types used for parsing and utilizing the [Trusted Setup]( https://ceremony.ethereum.org/) for the `KzgSettings`.
\ No newline at end of file
+2. `trusted_setup_points`: This module contains functions and types used for parsing and utilizing the [Trusted Setup]( https://ceremony.ethereum.org/) for the `KzgSettings`.
```
Comment (onbjerg) [inline documentation/src/crates/primitives/kzg.md:12]:
```suggestion
2. `trusted_setup_points`: This module contains functions and types used for parsing and utilizing the [Trusted Setup](https://ceremony.ethereum.org/) for the `KzgSettings`.
```

File: documentation/src/crates/interpreter/memory.md [hunk 3]
```diff
@@ -32,4 +32,4 @@ These opcodes change the memory:
 * CALL
 * CALLCODE
 * DELEGATECALL
-* STATICCALL
\ No newline at end of file
+* STATICCALL
```
Comment (onbjerg) [inline documentation/src/crates/interpreter/memory.md:35]: not sure what the diff is here?

**Review by onbjerg — CHANGES_REQUESTED** [review]

File: documentation/src/crates/interpreter/memory.md (hunk 3 above)
Comment (nnsW3) [reply documentation/src/crates/interpreter/memory.md]: just than with then

**Comment by nnsW3** [comment]
done

**Review by onbjerg — APPROVED** [review]

**Review by rakita — APPROVED** [review]
lgtm

**Pull request merged**